### PR TITLE
docs: fix incorrect guide on genesisBlock.json location

### PR DESCRIPTION
**Pull request by KaiserKarel**
## Proposed changes
I incorrectly described how to migrate the genesisBlock.json file. Fixed that and added descriptions on the other files in the crypto/src/networks/{NETWORK} directory.

### CODE REVIEW

**Comment by ghost** [comment]
@KaiserKarel Thanks for submitting this pull request, a maintainer will get back to you shortly!

**Comment by ghost** [comment]
@adrian69 @faustbrian @supaiku0 - please review this in the next few days. Be sure to explicitly select labels so I know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

File: docs/releases/v2.1/migrating_1.0_2.1.md [hunk 1]
```diff
@@ -164,12 +163,85 @@ module.exports = {
 };
 ```
 
-Your directory `MyNet` should now contain `genesisBlock.json`, `peers.json`, and a `plugins.js`. If you use the terms `mainnet`, `devnet` and `testnet` in your BridgeChain, you should create configuration directories for each of these networks, overriding the existing directories.
+Your directory `MyNet` should now contain `peers.json` and `plugins.js`. If you use the terms `mainnet`, `devnet` and `testnet` in your BridgeChain, you should create configuration directories for each of these networks, overriding the existing directories.
+
+### Crypto
+
+We also need to move our initial block definition, the genesisBlock, to the correct location. In previous versions it was stored alongside our other files, however, crypto related files now reside in `crypto/src/networks/{NETWORK}`. This directory contains the following four files:
```
Comment (faustbrian) [inline docs/releases/v2.1/migrating_1.0_2.1.md]: `packages/crypto/src/networks/{NETWORK}`

**Review by faustbrian — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@KaiserKarel Your pull request needs some changes. Please wait for a comment from one of our developers for more information.

**Review by kristjank — APPROVED** [review]

**Comment by ghost** [comment]
@KaiserKarel A member has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Review by faustbrian — APPROVED** [review]

**Comment by ghost** [comment]
@KaiserKarel A contributor has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
@KaiserKarel Your pull request has been merged but was not assigned a bounty tier. @adrian69 @faustbrian @supaiku0 - please assign a bounty tier to this pull request in the next few days.

**Pull request merged**